mw-webapp/src/logic/wayPage/wayStatistics/statisticWidget/StatisticWidget.module.scss

```diff
@@ -11,3 +11,7 @@
   width: 100%;
   justify-content: space-between;
 }
+
+.titleCheckboxIndex {
+  z-index: 1;
```

Comment:
let's use global variables for such purpose

Reply:
Why we need to use global variables? in global we have only 
![image](https://github.com/tritonJS826/masters-way/assets/112805668/a55a45fc-7fbb-4a3d-b978-ad93aaea2863)

Reply:
add to glibal variable